tests/commands/publish_tests.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from courseraprogramming.commands import publish
+
+from mock import ANY, call, MagicMock, patch
+
+
+class PublishParams:
```

Comment:
Previously, you included a trailing `()` after the class name. Here you elide them. Can you pick one and be consistent? (Unless it's a pep8 violation, I'd prefer how you do it here.)